cmd/podman/kube/play.go

```diff
@@ -171,6 +173,10 @@ func playFlags(cmd *cobra.Command) {
 
 	noTruncFlagName := "no-trunc"
 	flags.BoolVar(&playOptions.UseLongAnnotations, noTruncFlagName, false, "Use annotations that are not truncated to the Kubernetes maximum length of 63 characters")
+	_ = flags.MarkHidden(noTruncFlagName)
+	if err := flags.MarkDeprecated(noTruncFlagName, "podman kube complies to the Kubernetes annotation specifications"); err != nil {
+		logrus.Error("unable to mark --no-trunc flag deprecated")
```

Comment:
@mheon can you add this to your Deprecation list for Podman v5.* in RHEL notes please?